.gitignore

```diff
@@ -2,3 +2,4 @@
 .env
 *.test
 *.coverprofile
+/.idea
```

Comment:
What's this?

Reply:
Directory that https://www.jetbrains.com/go/ uses for its temporary files.